### PR TITLE
Wizard recipe: libexif-v0.6.21

### PR DESCRIPTION
**Pull request by innerlee**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: libexif
* Version: v0.6.21

@staticfloat please review and merge.

### CODE REVIEW

File: L/libexif/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:i686, libc=:musl),
+    Linux(:x86_64, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf),
+    MacOS(:x86_64),
+    Windows(:i686),
+    Windows(:x86_64)
+]
```
Comment (staticfloat) [inline L/libexif/build_tarballs.jl]: Does this not work on FreeBSD?  Is that the issue?

Comment (innerlee) [reply L/libexif/build_tarballs.jl]: there are compiling errors at freebsd and another platform. i don't know how to fix them.

**Comment by innerlee** [comment]
Freebsd does not build shared lib, it builds static lib only.

**Comment by giordano** [comment]
Another trick often needed: if the shared library for PowerPC is not built, you need to regenerate the `configure` script.  See https://github.com/JuliaPackaging/Yggdrasil/wiki/Build-Tricks#powerpc-linux

File: L/libexif/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+sources = [
+    "https://jaist.dl.sourceforge.net/project/libexif/libexif/0.6.21/libexif-0.6.21.tar.gz" =>
+    "edb7eb13664cf950a6edd132b75e99afe61c5effe2f16494e6d27bc404b287bf",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libexif-*/
+if [[ "${target}" == powerpc64le-* ]] || [[ "${target}" == *-freebsd* ]]; then
+    # Install `autopoint` and other tools needed by `autoreconf`
```
Comment (innerlee) [inline L/libexif/build_tarballs.jl]: can `autoreconf` etc. be pre-installed in the vm?

Comment (giordano) [reply L/libexif/build_tarballs.jl]: It's already available in the environment, not sure what you're referring to.

Comment (innerlee) [reply L/libexif/build_tarballs.jl]: sorry, the `autopoint`

Comment (giordano) [reply L/libexif/build_tarballs.jl]: I think this is the first time I needed to use this tool, I don't think it's worth adding it by default.

**Pull request merged**